tensorflow/workspace.bzl

```diff
@@ -99,6 +99,9 @@ def _execute_and_check_ret_code(repo_ctx, cmd_and_args):
 # Apply a patch_file to the repository root directory
 # Runs 'patch -p1'
 def _apply_patch(repo_ctx, patch_file):
+  if not repo_ctx.which("patch"):
```

Comment:
Do I make a mistake?  [`path repository_ctx.which()`](https://docs.bazel.build/versions/master/skylark/lib/repository_ctx.html#which) doesn't receive argument.

Reply:
Looks good to me.